draft-ietf-webtrans-http3.md

```diff
@@ -218,6 +218,27 @@ session have alternative mechanisms:
   limiting {{!RFC6585}}.  Unlike the previous method, this signal is directly
   propagated to the application.
 
+## Prioritization
+
+WebTransport does not provide any priority signalling mechanism for streams and
+datagrams within a WebTransport session; such mechanisms will be defined by
```

Comment:
```suggestion
datagrams within a WebTransport session; such mechanisms can be defined by
```

Reply:
+1

Reply:
or even just a "may be defined by other specifications", which would keep the door open to defining an actual WebTransport signal.